dvc/repo/experiments/utils.py

```diff
@@ -71,3 +71,15 @@ def remote_exp_refs_by_baseline(
         if ref.startswith(EXEC_NAMESPACE) or ref == EXPS_STASH:
             continue
         yield ExpRefInfo.from_ref(ref)
+
+
+def exp_commits(
+    scm: "Git", ref_infos: Iterable["ExpRefInfo"] = None
+) -> Generator[str, None, None]:
```

Comment:
I think, you can just do
```suggestion
) -> Iterator[str]:
```